docs/examples/pinecone-and-scaling.ipynb

```diff
@@ -310,16 +292,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Let's load our index first. As mentioned, `\"index\"` is the default index name, so we don't need to specify this parameter — but we do so below for demonstrative purposes."
+    "Let's load our index first. As mentioned, `\"index\"` is the default index name and `\"namespace\"` is the default namespace name, so we don't need to specify this parameter — but we do so below for demonstrative purposes."
```

Comment:
I was under the impression that the default namespace in Pinecone was the empty string "".
This seems to imply that the default namespace in semantic-router is "namespace".  Is that really what you are going for?